sass/_cta.scss

```diff
@@ -0,0 +1,13 @@
+.cta__wrapper {
+  @include component-box;
+  position: fixed;
+  top: 50%;
+  transform: translateY(-50%);
+  right: 0;
+  padding: 0;
+  z-index: 9999;
+  width: 3.75rem;
```

Comment:
possibly? when I checked out your branch and removed the extra width I didn't see a need for floats or see the buttons change position, and with the container being `fixed` it shouldn't default to 100% width. But if width prevents needing to use float go ahead and use width, if you're sure you need to